packages/ui/src/utils/proxyTypeRendererUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { ProxyType } from '../../types-and-hooks'
```

Comment:
Please see my comment in the issue, this is not future proof and having a dynamic way is better
https://github.com/ChainSafe/Multix/issues/79#issuecomment-1911936995